akka-http-jwt/src/main/resources/reference.conf

```diff
@@ -0,0 +1,33 @@
+#######################################
+# akka-http-jwt Reference Config File #
+#######################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka.http.jwt {
+  # Enables or disables the JWT signature validation.
+  # This is useful for development and testing purposes
+  # where you can still assert presence of claims without validating the signature.
+  dev = off
+
+  # The realm to use in the WWW-Authenticate header when a token is missing or invalid.
+  realm = "akka-http-jwt"
+
+  # Allows configuration for the JWT secrets used to verify tokens.
+  # An example config would be:
+  # secrets: [
+  #             {
+  #               key-id: my-key
+  #               issuer: my-issuer
+  #               algorithm: HS256
+  #               # The secret can be set via an environment variable or loaded from a file.
+  #               # To load the secret from an environment variables use:
+  #               secret: ${MY_PRECIOUS_SECRET}
+  #               # To load the secret from a file use (and remove the above secret setting):
+  #               secret-dir: /path/to/secrets
```

Comment:
@johanandren picking up from your comment [here](https://github.com/akka/akka-http/pull/4377#discussion_r1556080356):
> Pick up all/both the discrete paths from config rather than directory from config with hardcoded magic filenames.

If I’m understanding correctly.. you’re suggesting that instead of having a config for `secret-dir` we could have one where the complete file path is provided.. say: `secret-file=/path/to/secret.key` right?

But we would need different settings depending if its symmetric or asymmetric, right? For asymmetric keys one it would be something like:
`public-key=/path/to/public.key`
`private-key=/path/to/private.key`

Is this what you had in mind?

Reply:
Yes, I think depending on type we can just have different entries and read what is needed.

Reply:
tks, see https://github.com/akka/akka-http/pull/4377/commits/b2576e272027d54cd4e41dc2d7a2784aa44b96fd